sdk/core/azure-core/inc/azure/core/base64.hpp

```diff
@@ -10,8 +10,9 @@
 #pragma once
 
 #include <algorithm>
-#include <cstdint>
+#include <cstdint> // defines std::uint8_t
 #include <stdexcept>
+#include <stdint.h> // deprecated, defines uint8_t in global namespace.
```

Comment:
When do we include both in our code? What are the types or methods that trigger the issue?

Reply:
This issue applies to ALL types of the form `.int*_t` that have this issue (and a bunch of other random types defined in stdint.h), not just uint8_t (although that is the only type referenced in base64).

Theoretically this applies to any place we include cstdint, but this is the only one that customers are complaining about.

We need both of them because the integer types are deprecated in the global namespace. And apparently some C++ runtime libraries have followed through and removed the deprecated types from cstdint.

Anton has suggested (and I agree with him) that we should move all the integer types to use the std::xxx form. But for now, we can resolve the specific customer issue with a targeted fix.